backend/mockData/users.js

```diff
@@ -0,0 +1,20 @@
+import bcrypt from 'bcrypt';
+
+const saltRounds = 10;
+
+const mockedUsers = [
```

Comment:
Why did you use array here ?
Image you will have hundreds of users, it will take too much time to find right user, think about using object with such schema:
{
 [userId]: {
   userId: ...,
  ...otherUserData
}
}

Reply:
I knew that in the following homework I will have to connect and create database with all needed schemas and rewrite services' methods so they would work with db. So for mocking users just used an array because mockedUsers was just to test created endpoints and implemented authentication and was not intended to store actual users in the future.